src/altimate.ts

```diff
@@ -589,4 +602,10 @@ export class AltimateRequest {
       })}`,
     );
   }
+
+  async getHealthcheckConfigs() {
+    return this.fetch<DBTConfigResponse>(
+      `dbtconfig?${new URLSearchParams({ size: "100" }).toString()}`,
```

Comment:
size 100?

Reply:
the backend api is paginated which allows max value of page size to be 100. Current extension ui is not designed for pagination. In future we need to fix the UI but for now it should be fine.